js/Makefile

```diff
@@ -353,3 +353,8 @@ $(pkg_mod): $(pkg_desc)
 	mkdir -p $(pkg_mod)
 	touch $(pkg_mod)
 	touch node_modules/go.mod # avoid go to lookup in this directorys
+
+#### Debug
+
+print-%:
+	@echo $* = $($*)
```

Comment:
Do you want to keep it or did you forget to remove it?

Reply:
I want to keep it, I often use it for debugging, basically, each time we had issues with Makefile in the past, I used it and asked other people to run various `make print-Something` commands